src/main/java/org/spongepowered/mod/registry/SpongeGameRegistry.java

```diff
@@ -872,6 +874,14 @@ private void setWeathers() {
         }
     }
 
+    private void setTextActionFactory() {
+        try {
+            TextActions.class.getDeclaredField("factory").set(null, new SpongeTextActionFactory());
```

Comment:
Wouldn't this require ```setAccessible(true)``` in order to work? Surely it'll just fail silently because you're catching the exception?